tests/test_multithread.py

```diff
@@ -72,6 +72,21 @@ def create_remote_csv(fs, name, columns, colwidth, lines):
 
 
 @my_vcr.use_cassette
+def test_download_single_empty_file(tempdir, azure):
+    with azure_teardown(azure):
+        name = posix(test_dir, 'remote.csv')
+        lines = 0 # the file should have no bytes in it
+        size, checksum = create_remote_csv(azure, name, 10, 5, lines)
+        fname = os.path.join(tempdir, 'local.csv')
+
+        # single chunk
+        down = ADLDownloader(azure, name, fname, 1, size + 10, overwrite=True)
+        assert md5sum(fname) == checksum
+        assert os.stat(fname).st_size == size
+        assert linecount(fname) == lines
+        os.remove(fname)
```

Comment:
:) I bring it up as I have found it easy to accidentally check in test files from test suites and not notice until later.